core/capabilities/compute/transformer.go

```diff
@@ -82,9 +87,16 @@ func (t *transformer) Transform(req capabilities.CapabilityRequest, opts ...func
 		if err != nil {
 			return capabilities.CapabilityRequest{}, nil, NewInvalidRequestError(err)
 		}
+		if td < 0 || td > t.config.MaxTimeout {
```

Comment:
Should a timeout duration of 0 valid here?

Reply:
Yes good catch :)